docs/SpiraApps/QuickTasks.md

```diff
@@ -0,0 +1,46 @@
+# Quick Tasks SpiraApp
+
+!!! warning "This SpiraApp's functionality requires SpiraTeam or SpiraPlan to work."
+
+This SpiraApp lets users create, complete, and view tasks from a convenient Product Home Page widget. 
+
+## Setup
+To effectively implement Quick Tasks in your product, you have to set up the Task Type that will be used and make it use a specific workflow. 
+
+### Product Template Setup
+You need to configure each product template for the product(s) you want to use Quick Tasks:
+
+1. Go to Tasks > Workflow and add a new Workflow called "Quick Task Workflow". Do not change anything else about this workflow.
+
+2. Go to Tasks > Type and add a new Type called "Quick Task" (or other name of your choice) and select the Workflow you just created
```

Comment:
First, delete empty lines between list items - not required. Empty lines are only required between paragraphs in markdown.

Second, is this required or optional? I think again it is optional